src/Cassandra.IntegrationTests/TestClusterManagement/TestClusterManager.cs

```diff
@@ -70,15 +73,18 @@ public static Version CassandraVersion
                         // C* 3.0
                         return Version3Dot0;
                     }
-                    if (dseVersion < Version6Dot0)
+                    if (dseVersion <= Version6Dot9)
```

Comment:
Hmm this is a tricky change. Technically DSE 6.x is in between 3.11 and 4.0 (it has some 4.0 features). This might result in some DSE 6.x tests not running because we declared the test as compatible with C* 4.0 and we relied on it working this way to run for DSE as well. Maybe try to check the total test count per server version before and after the changes to make sure we're not losing tests with these changes.

Reply:
Before the fix, dse-6.7.17 net6: total tests 1145, skipped 108.
After the fix, dse-6.7.17 net6: total tests 1145, skipped 107.

Before fix, dse-6.8 net6, skipped 28
After fix, dse-6.8 net6, skipped 27

So after my fix, it actually skips fewer tests. I think it may be because my change in `CheckCassandraVersion`. Previously it maps C* version to DSE version when comparing, using `GetDseVersionFromCassandraVersion`. I changed it so it maps DSE version to C* version instead. I think it makes more sense, and align with other drivers better, too.

Reply:
> Before fix, dse-6.8 net6, skipped 28
> After fix, dse-6.8 net6, skipped 27

Do you have a total number for `dse-6.8 net6` similar to `dse-6.7.17 net6`?

Reply:
dse-6.8.30

Reply:
I mean total number of tests, you wrote the total number of tests dse-6.7 but not dse-6.8

Reply:
Oh the same, the number of total tests is 1145. This is the number of tests before skipping any, so it will be the same for any versions, before and after fix.

Reply:
The one less skip count is about `Should_FailFast_When_PreparedStatementIdChangesOnReprepare`, because "This test relies on a bug that is fixed in this server version." I assume that test does not matter in this case.
Additionally, I just found out that `[TestCassandraVersion(x,x)]` never maps DSE version to C* OSS version. Say if you say `[TestCassandraVersion(5, 0)]` and the executing version is DSE-6.7, then 6.7 > 5.0, so the test will proceed to run.
Do we agree that
1. We should map DSE 6.x to C* 3.11, and
2. We change `[TestCassandraVersion(x,x)]` to map DSE version to C* version before it compares?